### PR TITLE
Blog post for KubeCon NA 2022

### PR DESCRIPTION
**Pull request by deepeshaburse**
**Description**

Blog post for KubeCon + CloudNativeCon North America 2022.

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 8a83c3102f7068c21886d56c483c6737e2a016a8 at: https://634f945245c10e56be7a7cc7--layer5.netlify.app

File: src/collections/blog/2022/2022-10-19-kubeconna2022/index.mdx [hunk 1]
```diff
@@ -0,0 +1,99 @@
+---
+title: Layer5 at KubeCon + CloudNativeCon North America
+subtitle: Join us at KubeCon + CloudNativeCon North America 2022
+date: 2022-01-14 10:30:05 -0530
```
Comment (leecalcote) [inline src/collections/blog/2022/2022-10-19-kubeconna2022/index.mdx:4]:
```suggestion
date: 2022-10-19 08:00:00 -0530
```

Today's date.

**Comment by l5io** [comment]
🚀 Preview for commit 637d7c2fa6ad46b958b4dcaacd1e0c97c541d235 at: https://634ffb9c7c99960fdac00765--layer5.netlify.app

File: src/collections/blog/2022/2022-10-19-kubeconna2022/index.mdx [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+<h2>Resumé Writing Workshop</h2>
+<p>
+Attendees are encouraged to join this unique DEI event, which will feature a resume writing class, mingling, networking, and refreshments. The event will take place 
+on October 25 from 4:00 PM to 6:00 PM and is sponsored by Visit Detroit. Scholarship recipients and regional groups that support diversity, equality, and inclusion 
+in technology have received special invitations.
+</p>
+
+<h2>Layer5 at KubeCon + CloudNativeCon North America</h2>
+<h3>October 26, 2022</h3>
```
Comment (leecalcote) [inline src/collections/blog/2022/2022-10-19-kubeconna2022/index.mdx:62]:
```suggestion
<h4>October 26, 2022</h4>
```

File: src/collections/blog/2022/2022-10-19-kubeconna2022/index.mdx [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+<h3>Project Office Hours on Meshery</h3>
```
Comment (leecalcote) [inline src/collections/blog/2022/2022-10-19-kubeconna2022/index.mdx:63]:
```suggestion
<h3>Meshery Project Office Hours</h3>
```

File: src/collections/blog/2022/2022-10-19-kubeconna2022/index.mdx [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+ <ul>
+   <li> Applications </li>
+   <li> Filters </li>
+   <li> Patterns </li>
+   <li> Dynamic Management UIs </li>
+  </ul>
+  </p>
+
+<h3>October 27, 2022</h3>
+<h3>Project Office Hours on Service Mesh Performance</h3>
```
Comment (leecalcote) [inline src/collections/blog/2022/2022-10-19-kubeconna2022/index.mdx:79]:
```suggestion
<h4>October 27, 2022</h4>
<h3>Service Mesh Performance Project Office Hours</h3>
```

**Pull request merged**